share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -137,6 +144,36 @@ macro(rocm_create_package)
     include(CPack)
 endmacro()
 
+function(rocm_get_patch_version OUTPUT)
```

Comment:
This redefines the `rocm_get_patch_version`, you should just use the version in ROCMSetupVersion.cmake. Just add a `include(ROCMSetupVersion)`.

Reply:
done.